.github/workflows/release.yml

```diff
@@ -168,10 +164,23 @@ jobs:
       with:
         name: azureauth-${{ github.event.inputs.version }}-osx-arm64.tar.gz
         path: azureauth-${{ github.event.inputs.version }}-osx-arm64.tar.gz
+      
+  linux_release:
+    runs-on: ubuntu-latest
+    needs: [validate]
+    steps:
+    - name: Build, Sign and Publish Linux binaries
+      run: python .\bin\trigger_azure_pipelines.py

```

Comment:
These are backslashes and I think you're gonna find that gets weird (probably doesn't work as you expect) on Linux. Try `/` instead.